outbox/shipment-service/src/main/java/io/debezium/examples/outbox/shipment/messagelog/ConsumedMessage.java

```diff
@@ -3,20 +3,18 @@
  *
  * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.debezium.examples.outbox.shipment.log;
+package io.debezium.examples.outbox.shipment.messagelog;
 
 import java.time.Instant;
 import java.util.UUID;
 
-import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
 
 @Entity
 public class ConsumedMessage {
 
     @Id
-    @Column(columnDefinition = "BINARY(16)")
```

Comment:
This one created an error for my when Hibernate created the schema in Postgres. It was needed for MySQL/MariaDB, but Postgres supports UUID, so I could just remove it.

Reply:
But this does bring up an interesting question regarding the outbox table in the extension.  We store a UUID field there and I am wondering if we should consider allowing the user to define a custom column definition configuration option we plug-in here if supplied?

Reply:
So about that one, do you want to leave that for the follow-up? If so, could you update the issue [DBZ-1701](https://issues.redhat.com/browse/DBZ-1701) accordingly?

Reply:
I think that'd probably best as I want to do some more testing across various data sources.  I actually logged the follow-ups in [DBZ-1711](https://issues.redhat.com/browse/DBZ-1711) since the other you referenced seemed to be related to CloudEvents.

Reply:
Hah, yeah, was too late in the day for me when I wrote that comment. Thanks for logging a new one!